.gitignore

```diff
@@ -3,3 +3,9 @@ datasets/__pycache__/
 eval/__pycache__/
 models/__pycache__/
 ops/__pycache__/
+
+.github/
+datasets/data/archive/
```

Comment:
Nit: Ignoring the folder itself is enough
```suggestion
datasets/
```

Reply:
Ah, yes. Thanks (: